src/komodo_defs.h

```diff
@@ -46,7 +46,7 @@
     // 7113400 = 5x current KMD blockheight. 
 // to add 4th season, change NUM_KMD_SEASONS to 4, and add timestamp and height of activation to these arrays. 
 
-#define NUM_KMD_SEASONS 6
+#define NUM_KMD_SEASONS 5
```

Comment:
Why it's back to 5 again?

Reply:
I had accidentally merged my ARRR-HLTC-debug into the branch being PR'd here. It's already reverted.